server/routes/connections.js

```diff
@@ -3,10 +3,12 @@ import { Router } from 'express';
 
 import multipartRequest from '../lib/multipartRequest';
 
+const CONNECTIONS_LIST_LIMIT = 50;
```

Comment:
If the only thing we fetch connections for is the dropdown for adding a new user, and we only show up to 20 in that drop down as per [here](https://github.com/auth0-extensions/auth0-delegated-administration-extension/pull/292/files#diff-5d884cca10d6104c257713d1a0ad00def896f57384d897d4b57dd3d41b4b8892R3), is there any need to fetch more than 20 from the db? Should we re-use the same const and only fetch 20 connections?

Reply:
to answer this, I think this number is set at 50 as it's the same as the `perPage` number, so we're fetching one full page of connections. Reducing this to 20 doesn't really result in perf improvements as it's still one API2 call.